service/step_handlers_test.go

```diff
@@ -618,7 +618,7 @@ func handleQueryVolumeIDByKey(w http.ResponseWriter, r *http.Request) {
 		writeError(w, "induced error", http.StatusRequestTimeout, codes.Internal)
 		return
 	}
-	req := new(types.VolumeQeryIdByKeyParam)
```

Comment:
We need to change it to "VolumeQeryIDByKeyParam"
This is because in goscaleio, we changed it from 
VolumeQueryIdByKeyParam -> VolumeQeryIDByKeyParam; due to a complaint from golint.  
Change in GoScaleio can be viewed [here](https://github.com/dell/goscaleio/pull/11/files#diff-70a1e59eac932a2eba4fad961a903351b51e265a32fab17a0520a25d9a0b0278R454).